Mathlib/Topology/ContinuousOn.lean

```diff
@@ -861,6 +861,10 @@ theorem ContinuousOn.comp_continuous {g : β → γ} {f : α → β} {s : Set β
   rw [continuous_iff_continuousOn_univ] at *
   exact hg.comp hf fun x _ => hs x
 
+theorem ContinuousOn.image_comp_continuous {g : β → γ} {f : α → β} {s : Set α}
+    (conton : ContinuousOn g (f '' s)) (cont : Continuous f) : ContinuousOn (g ∘ f) s :=
```

Comment:
I think naming the hypotheses `hg` and `hf`, respectively, would be more in line with the other lemmas in the file.